google-cloud-bigquery/lib/google/cloud/bigquery/schema.rb

```diff
@@ -219,13 +219,12 @@ def load source
             source.rewind
             schema_json = String source.read
           elsif source.is_a? Array
-            schema_json = source.to_json
+            schema_json = JSON.dump source
           else
```

Comment:
Now that `JSON.dump` is being called here, the require should be moved to this file.